code/modules/cargo/exports/food_and_drink.dm

```diff
@@ -0,0 +1,18 @@
+/**
+ * Original Food export file got eaten somewhere along the line and I have no idea when or where it got completely deleted.
+ * Foods given a venue value are exportable to cargo as a backup to selling from venues, however at the expense of elasticity.
+ */
+/datum/export/food
+	cost = 10
+	unit_name = "serving"
+	message = "of food"
+	export_types = list(/obj/item/food)
+	include_subtypes = TRUE
+	exclude_types = list(/obj/item/food/grown)
+
+/datum/export/food/get_cost(obj/O, allowed_categories, apply_elastic)
+	. = ..()
```

Comment:
Why bother calling parent if you disregard the parent call in this case? I believe the base `get_cost()` returns a value based on cost which doesn't matter here